recipes/perl4-corelibs/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "perl4-corelibs" %}
+{% set version = "0.0.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name|lower }}_{{ version }}.tar.gz
+  url: http://search.cpan.org/CPAN/authors/id/Z/ZE/ZEFRAM/Perl4-CoreLibs-0.003.tar.gz
+  md5: f74747c234470a087871f564a5f5db15
+
+build:
+  number: 0
+  script: "cpanm ."
+
+requirements:
+  build:
+    - perl-threaded
+    - perl-app-cpanminus
+    - perl-module-build
+
+  run:
+    - perl-threaded
+
+
+test:
+  commands:
+    - perl -e 'require "getopts.pl"' 
```

Comment:
Can you not use something like this? https://github.com/bioconda/bioconda-recipes/pull/3362/files#diff-15ac499c35dfd349ebcdf8aa6252f868R41